### ISSUE TITLE
Formalize paradigms of actions

### ISSUE THREAD

**Issue by PlagaMedicum** [ostis-apps/ostis-linguistics#10]
We need to formalize paradigms of actions in A.Hardzei "Theory for automatic generation of knowledge architecture: TAPAZ-2" book.

**Comment by qian2020** [comment]
it's about the sematic of verb?

**Comment by PlagaMedicum** [comment]
> it's about the sematic of verb?


Yes, in general. There was added a section with [this PR](https://github.com/ostis-apps/ostis-linguistics/pull/19), but work with this still not completed and the section requires improvements and fixes.

**Comment by PlagaMedicum** [comment]
Need to return to this subjdomain later and fix the classification
